src/components/Stream.tsx

```diff
@@ -39,6 +47,16 @@ export const Stream = () => {
   const [streamState, setStreamState] = useState(StreamState.Unset)
   const { file } = useRouteLoaderData(PATHS.FILE) as IndexLoaderData
 
+  const contextMenuGuard = useCallback(
+    (e: globalThis.MouseEvent) => {
+      return (
+        sceneInfra.camControls.wasDragging === false &&
+        btnName(e).right === true
+      )
+    },
+    [sceneInfra.camControls.wasDragging]
+  )
+
```

Comment:
How come we want to "refresh" the callback on `wasDragging` changes? I think `useCallback` may be unnecessary here

Reply:
Right, I'd just do contextMenuGuard = myFunc (not necessary tho for the PR)

Reply:
Nope you're right, I don't need `useCallback`. Thanks!